cardano_node_tests/tests/tests_conway/test_info.py

```diff
@@ -239,3 +239,96 @@ def test_info(
             votes=governance_utils.VotedVotes(cc=votes_cc, drep=votes_drep, spo=votes_spo),
             txhash=vote_txid,
         )
+
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.testnets
+    @pytest.mark.smoke
```

Comment:
We mark tests as "smoke" if they can finish under 2 mins. This one needs to wait for epoch boundary, so it is not a smoke test by our definition.

Reply:
Thank you for the information. It has been integrated into the test_info test itself.